pyproject.toml

```
@@ -39,6 +39,9 @@ wheel = "wheel.cli:main"
 bdist_wheel = "wheel.bdist_wheel:bdist_wheel"
 
 [project.optional-dependencies]
+convert = [
+    "setuptools",
```

Should this be pinned?

```suggestion
    "setuptools >= 65",
```